src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -307,32 +257,41 @@ function setupRecordingGainVis() {
   canvasContext.fillRect(0, 0, 1, 1);
 
   let currentX = 0;
+  let previousY = height / 2;
+  // Adjust the amplitude value to increase or decrease the size of the waveform
+  const amplitude = height * 2;
 
   function draw(currentSampleGain) {
-    const centerY = ((1 - currentSampleGain) * height) / 2;
-    const gainHeight = currentSampleGain * height;
+    const centerY = height / 2 - currentSampleGain * amplitude;
 
     // Clear current Y-axis.
```

Comment:
I don't find these comments (around `canvas`) are useful or meaningful. Let's get rid of them.